spec/models/labels/plate_label_cellaca_qc_spec.rb

```diff
@@ -18,14 +18,15 @@
   describe '#qc_label_definitions' do
     subject(:qc_label_definitions) { label.qc_label_definitions }
     let(:label) { described_class.new(labware) }
+    let(:date_format) { /\A\s?\d{1,2}-[A-Z]{3}-\d{4}\z/ } # e.g., ' 4 JUL 2023' or '24 JUL 2023'
```

Comment:
I accept that this regex matches the intended formats, but it also matches, for example `' 12-JUL-2023'` when you didn't intend a space before the two digits.  I don't know if this is actually important, but a regex that only allows the space for a single digit date is `/\A(?:\s\d|\d\d)-[A-Z]{3}-\d{4}\z/`

Your examples don't actually match your regex which expects dashes and not spaces between the segments.

Reply:
Sure, I actually ripped this regex from another test, I think some of the other tests check the exact equals using the same datetime string from https://github.com/sanger/limber/blob/develop/app/models/labels/base.rb#L35. That might be the nicer way of doing it.